lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/persistence/DBOMembershipInvitationTest.java

```diff
@@ -139,43 +139,6 @@ public void testTranslator() throws Exception {
 		assertEquals(dto, MembershipInvitationUtils.copyDboToDto(dbo));
 	}
 
-	@Test
-	public void testTranslatorRefactor() throws IOException {
-		DBOMembershipInvitation backup = new DBOMembershipInvitation();
-		String oldProperties =
-				"<MembershipInvtnSubmission>\n" +
-				"  <createdOn>2017-10-30 15:54:48.793 UTC</createdOn>\n" +
-				"  <message></message>\n" +
-				"  <id>9603838</id>\n" +
-				"  <createdBy>273995</createdBy>\n" +
-				"  <inviteeId>3341743</inviteeId>\n" +
-				"  <teamId>3320424</teamId>\n" +
-				"</MembershipInvtnSubmission>\n";
-		backup.setProperties(zip(oldProperties.getBytes()));
-
-		// Method under test
-		DBOMembershipInvitation translated = backup.getTranslator().createDatabaseObjectFromBackup(backup);
-
-		String translatedProperties = new String(unzip(translated.getProperties()));
-		// Assert that the top level xml tags are updated
-		assertTrue(translatedProperties.startsWith("<MembershipInvitation>"));
-		assertTrue(translatedProperties.endsWith("</MembershipInvitation>"));
-
-		String expectedProperties =
-				"<MembershipInvitation>\n" +
-				"  <createdOn>2017-10-30 15:54:48.793 UTC</createdOn>\n" +
-				"  <message></message>\n" +
-				"  <id>9603838</id>\n" +
-				"  <createdBy>273995</createdBy>\n" +
-				"  <inviteeId>3341743</inviteeId>\n" +
-				"  <teamId>3320424</teamId>\n" +
-				"</MembershipInvitation>\n";
-		MembershipInvitation expectedDTO = deserialize(zip(expectedProperties.getBytes()));
-		MembershipInvitation translatedDTO = deserialize(translated.getProperties());
-		// Assert that the translated DTO's contents are identical to the old DTO's
-		assertEquals(expectedDTO, translatedDTO);
-	}
-
 	@Test
 	public void testTranslatorRefactorAlreadyTranslated() throws IOException {
```

Comment:
I believe you can now remove ALL tests of the translator, since the translator is now trivial.